components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -2096,6 +2123,46 @@ private void parseOpenIDConnectConfig(OMElement oauthConfigElem) {
         }
     }
 
+    /**
+     * Parse supported encryption algorithms set and add them to supportedIdTokenEncryptionAlgorithms.
+     * @param algorithms   OMElement of supported algorithms.
+     */
+    private void parseSupportedIdTokenEncryptionAlgorithms(OMElement algorithms) {
```

Comment:
Usually there's a new line added after the method signature for clarity. If you have applied appropriate formatter it should be formatted so by default.

Reply:
Changed.